MapboxAndroidDemo/build.gradle

```diff
@@ -12,8 +12,8 @@ android {
     applicationId "com.mapbox.mapboxandroiddemo"
     minSdkVersion 15
     targetSdkVersion 25
-    versionCode 38
-    versionName "5.0.0-beta.3"
+    versionCode 50
```

Comment:
Why'd the version code get bumped to somewhat match the versionName? is there any advantage to this?